Makefile

```diff
@@ -1,6 +1,19 @@
-install: 
-	mkdir -p /usr/share/gnome-background-properties/
-	install -D -m 0644 wallpapers-leap15.xml ${DESTDIR}/usr/share/gnome-background-properties/wallpapers-leap15.xml
-	mkdir -p /usr/share/wallpapers/leap15/
-	install -D -m 0644 leap15/*.jpg ${DESTDIR}/usr/share/wallpapers/leap15/
-	install -D -m 0644 leap15/*.xml ${DESTDIR}/usr/share/wallpapers/leap15/
+VERSION = 16
+SVG_FILES = $(wildcard leap$(VERSION)/*.svg)
```

Comment:
How about we install both directories and split them in the spec?

Reply:
We could split the older ones to the -retro subpackage.

Reply:
But it will *significantly* increase build time with optipng -o5 ... unless we don't do that on build time.